app/src/androidTest/java/com/github/se/project/ui/lesson/AddLessonTest.kt

```diff
@@ -114,9 +152,19 @@ class AddLessonTest {
     // Select location
     composeTestRule.onNodeWithTag("mapButton").performClick()
     composeTestRule.onNodeWithTag("mapContainer").performClick()
-    Thread.sleep(2000) // Wait for the map to load
-    val device = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
-    device.click(device.displayWidth / 2, device.displayHeight / 2)
+
+    // replace the following code with the composeTestRule equivalent as
+    // the Thread.sleep() method is not recommended and
+    // device.click() is not well supported in compose
```

Comment:
This comment is not completely usefull since it explains why the code has changed and not really how it works, maybe we could remove it.